README.rst

```diff
@@ -185,7 +185,7 @@ Check out the `Authentication section`_ in our documentation to learn more.
 You may also find the `authentication document`_ shared by all the
 ``google-cloud-*`` libraries to be helpful.
 
-.. _Authentication section: https://google-cloud-python.readthedocs.io/en/latest/core/auth.html
+.. _Authentication section: https://google-cloud.readthedocs.io/en/latest/core/auth.html
```

Comment:
Thanks for the patch!  I think the preferred / canonical location for these docs is now https://googleapis.dev/python/google-api-core/latest/auth.html